### ISSUE TITLE
Specific use of play data

### ISSUE THREAD

**Issue by COST-97** [bytedance/GR-MG#10]
Thank you for your excellent work!

I did the following comparative experiments: (1) training with only the first 10% of ABC data with language annotations after shuffle; (2) training with the first 10% of ABC data with language annotations and play data, and then fine-tuning on the first 10% of ABC data with language annotations after shuffle.

The test results show that the performance of the second method seems to be significantly lower than the first method.

Could you share your training process in detail? For example, which 10% of data is used, how many steps of training and fine-tuning are used in the second method, and other things worth noting?

Thank you very much!

**Comment by LPY1219** [comment]
Hi,
For the second setting, I sampled the first 10% of the ABC data from each task. Initially, I trained the policy using only play data. Subsequently, I fine-tuned the model on the sampled ABC data.

**Comment by COST-97** [comment]
Hello:
Glad to receive your reply.
Could you share the code related to “the first 10% of the ABC data from each task” and the training script file, hyperparameter configuration file? Are the hyperparameters for training with play data and 10% ABC data the same? For example, a smaller learning rate, and fewer training times?

Thanks again for your excellent work and timely reply.

**Comment by LPY1219** [comment]
Hi,

In the `CalvinDataset_Policy` class, the `task_num `parameter is used to specify the maximum number of trajectories sampled from each task. You can adjust this parameter to obtain any percentage of the data you need.

I’ve used the same training parameters as in the full dataset configuration.
